app/helpers/feedback_helper.rb

```diff
@@ -0,0 +1,11 @@
+module FeedbackHelper
+  def current_user_email(current_jobseeker, current_publisher)
+    if current_jobseeker
```

Comment:
this could be:

`current_jobseeker&.email.presence || current_publisher&.email.presence`